ocaml-lsp-server/src/position.ml

```diff
@@ -1,6 +1,15 @@
 open Import
 include Lsp.Types.Position
 
+let start = { line = 1; character = 1 }
+
+let end_of_string s =
+  let lines = String.split_lines s in
+  let length_last_line =
+    ListLabels.nth lines (List.length lines - 1) |> String.length
```

Comment:
`List.last` is already available and is safer to use.

Reply:
Actually, I don't see `end_of_string` being used anywhere anyway.

Reply:
I removed it